build/Jenkinsfile

```diff
@@ -20,19 +20,39 @@ pipeline {
     stages {
         stage('Install') {
             steps {
-                sh 'make node_modules'
+                ansiColor('xterm') {
+                    sh 'make node_modules'
+                }
             }
         }
 
         stage('Check Style') {
             steps {
-                sh 'make check-style'
+                ansiColor('xterm') {
+                    sh 'make check-style'
+                }
+            }
+        }
+
+        stage('Check i18n') {
+            steps {
+                ansiColor('xterm') {
+                    sh """
+                        cp i18n/en.json /tmp/en.json
+                        mkdir -p /tmp/fake-mobile-dir/assets/base/i18n/
```

Comment:
@jespino do we need the fake-mobile-dir?

Reply:
Well, is the simplest way to solve this. The idea is, when I extract I use both repos to have all the information, but for this case I only need one of them, and this is the simplest approach to achieve that.